ui-admin/src/study/surveys/VersionSelector.tsx

```diff
@@ -27,7 +27,7 @@ export default function VersionSelector({ studyShortname, stableId, updateVersio
           { versionList.map(survey => {
             return <li key={survey.version}>
               <button className="btn btn-secondary" onClick={() => updateVersion(survey.version)}>
-                {survey.version} <span className="detail">({survey.createdAt?.substring(0, 16)})</span>
+                {survey.version} {/*<span className="detail">({survey.createdAt?.substring(0, 16)})</span>*/}
```

Comment:
createdAt was previously incorrectly typed as string instead of number. Left this commented out for now since this entire component is broken (it requests data from an API endpoint that doesn't exist).